test/runtests.jl

```diff
@@ -68,29 +68,31 @@ end
 @testset "Mismatch" begin
     mdutils = nothing
     devlist = nothing
-
-    for imsz in ((7,10), (6,5))
-        for maxshift in ((4,3), (3,2))
-            Apad = parent(ImageFiltering.padarray(reshape(1:prod(imsz), imsz[1], imsz[2]), Fill(0, maxshift, maxshift)))
-            Bpad = parent(ImageFiltering.padarray(rand(1:20, imsz[1], imsz[2]), Fill(0, maxshift, maxshift)))
-            for RM in RMlist
-                # intensity normalization
-                mm = RM.mismatch(Apad, Bpad, maxshift, normalization=:intensity)
-                num, denom = RegisterCore.separate(mm)
-                mmref = CenterIndexedArray{Float64}(undef, (2 .* maxshift .+ 1)...)
-                for j = -maxshift[2]:maxshift[2], i = -maxshift[1]:maxshift[1]
-                    Bshift = circshift(Bpad,-[i,j])
-                    df = Apad-Bshift
-                    mmref[i,j] = sum(df.^2)
+    for threading in (true, false)
```

Comment:
Since these are really two separate tests I think this should be lifted:
```
@testset "Mismatch (threading=$threading)" for threading in (true, false) ...
```